### PR TITLE
Ports a ZAS CSRFZ improvement.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Ports https://github.com/Baystation12/Baystation12/pull/32291.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by JohnWildkins — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Comment by Kapu1178** [comment]
Hi. Please give proper changelog credit to myself and Lohikar.

**Comment by NonQueueingMatt** [comment]
> Hi. Please give proper changelog credit to myself and Lohikar.

You're right, I just musclememory'd my name in. My bad.

File: html/changelogs/mattatlas-csrfz.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: MattAtlas
```
Comment (NonQueueingMatt) [inline html/changelogs/mattatlas-csrfz.yml:30]:
```suggestion
author: Kapu1178, Lohikar, MattAtlas
```

**Comment by JohnWildkins** [comment]
> Hi. Please give proper changelog credit to myself and Lohikar.

thank you for your service 🙇‍♂️

**Pull request merged**